lib/easyloggingpp/easylogging++.cc

```diff
@@ -1,17 +1,13 @@
 //
 //  Bismillah ar-Rahmaan ar-Raheem
 //
-//  Easylogging++ v9.96.7
+//  Easylogging++ v9.97.1
```

Comment:
Migration to new version of the used library is a significant fact. I'll try not to forget mentioning it in final commit message upon merge, but it would be good to see it in future namings for pull requests

Reply:
Agreed.